pkgs/default.nix

```diff
@@ -15059,6 +15060,7 @@ directory
 ekg-core
 ether
 exceptions
+extra
```

Comment:
Not sure where this comes from. Not sure it matters much either. Think it's already on `develop` too...